.ci/run.sh

```diff
@@ -75,7 +75,6 @@ function run_plugin() {
   fi
 
   if [ "x${GITHUB_ACTIONS}" == "xtrue" ] && [ "x${GITHUB_REF}" == "xrefs/heads/master" ]; then
-    git status
```

Comment:
Did you fetch origin before you did the diff? That was an important part...

Reply:
You might want to do `git log --walk-reflogs repo_predict` and checkout the one before the reset to master